iOSClient/Viewer/NCViewerMedia/NCViewerMedia.swift

```diff
@@ -92,9 +92,26 @@ class NCViewerMedia: UIViewController {
             statusViewImage.image = nil
             statusLabel.text = ""
         }
+        
+        if metadata.classFile == NCCommunicationCommon.typeClassFile.video.rawValue || metadata.classFile == NCCommunicationCommon.typeClassFile.audio.rawValue {
 
-        playerToolBar.viewerMediaPage = viewerMediaPage
+            playerToolBar = Bundle.main.loadNibNamed("NCPlayerToolBar", owner: self, options: nil)?.first as? NCPlayerToolBar
+            if let playerToolBar = playerToolBar {
+                view.addSubview(playerToolBar)
+                playerToolBar.translatesAutoresizingMaskIntoConstraints = false
+                playerToolBar.topAnchor.constraint(equalTo: view.topAnchor, constant: 0).isActive = true
+                playerToolBar.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: 0).isActive = true
+                playerToolBar.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 0).isActive = true
+                playerToolBar.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: 0).isActive = true
```

Comment:
good 

> This breaks #1882 again..
> 
> **UX**: Subtitle button acts as a toggle. I think it might be clearer to do it similar to native iOS subtitles and have a separate "Off" subtitle..?
> 
> ### Notes:
> * doesn't support embedded subtitles(?)
> * subtitles don't support [formatting](https://en.wikipedia.org/wiki/SubRip#Formatting)
> * subtitles need to be downloaded beforehand. Alternatively, could show an alert like: "Subtitle file(s) detected, download?"

This is only the first implementation , there will be future improvements..

Reply:
> Hmm.. why not use `view.safeAreaLayoutGuide` here?

because do not have effects .. but changed.

Reply:
Yes, the Author is Federico (Correct)